src/native/asm.gypi

```diff
@@ -2,14 +2,14 @@
 {
     'conditions': [
 
-        [ 'node_arch=="x64"', {
-            'defines': ['HAVE_AS_KNOWS_AVX512', 'HAVE_AS_KNOWS_SHANI'],
-        }],
-
         # LINUX
         [ 'OS=="linux"', {
             'conditions': [
                 [ 'node_arch=="x64"', {
+                    'defines': [
+                        'HAVE_AS_KNOWS_AVX512',
+                        'HAVE_AS_KNOWS_SHANI',
+                    ],
```

Comment:
in addition to this AVX512 fix we just need another tiny fix so it can run on Mac M1 with rosetta (as x64) - there's this unwanted space that creeped in at the end of the filename string here. Fix it too and we'll be able to run it locally...
https://github.com/noobaa/noobaa-core/blob/bbb342f02571b6c6ae4fc89fcc7a5a19004d5c79/src/native/third_party/isa-l.gyp#L96